requirements.txt

```diff
@@ -19,6 +19,7 @@ Jinja2>=2.10.1,<3.0
 kubernetes==10.0.1
 misaka==2.1.1
 model_mommy==1.6.0
+moto==1.3.13
```

Comment:
Isn't this more of a `requirements.dev.txt` dependency? Does the real thing rely on this to build/deploy?

Reply:
Tests are in the docker image, because we run tests with docker-compose. I would like to remove the tests from the image, though...

Reply:
We don't run the tests during the Docker image build. I would like to, but the tests require a database to run against. We could build the image with tests included, run the tests and then build and push another image from the same codebase? Not in scope of this PR, though.